tests/PHPStan/Parser/PhpDocParserTest.php

```diff
@@ -1518,10 +1576,9 @@ public function provideMultiLinePhpDocData(): array
 							new IdentifierTypeNode('Foo'),
 							false,
 							'$foo',
-							'1st multi world description'
+							'1st multi world description some text in the middle'
```

Comment:
This has no additional line breaks, so it is considered a continuation of the parameter description.